src/utils/private.ts

```diff
@@ -161,6 +161,13 @@ export const getWideConfig = async (context: Context) => {
     openAIKey: mergedConfigData.openAIKey,
     openAITokenLimit: mergedConfigData.openAITokenLimit,
     staleBountyTime: mergedConfigData.staleBountyTime,
+    startCommandEnabled: mergedConfigData.startCommandEnabled,
```

Comment:
Why do we have a separate `startCommandEnabled` if we already have `commandSettings`?

Reply:
Can You provide more context?

`startCommandEnabled` is for /start as it makes more sense than `assignCommandEnabled`

Reply:
> Can You provide more context?
> 
> `startCommandEnabled` is for `/start` as it makes more sense than `assignCommandEnabled`

Reply:
Can You provide more context?

 `startCommandEnabled` is for `/start` as it makes more sense than `assignCommandEnabled`

Reply:
`commandSettings` contains the `/start` command and you can enable or disable it there.